extensions/wikia/FacebookClient/FacebookClientController.class.php

```diff
@@ -77,33 +78,36 @@ public function deauthorizeCallback() {
 			$log->info( 'Deauthorization callback received with invalid signature', [
 				'method' => __METHOD__,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "method" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)